docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -818,6 +818,19 @@ Please see the xref:security-openid-connect-client-reference.adoc#token-propagat
 [[integration-testing]]
 === Testing
 
+[[dev-mode]]
+==== {extension-name} Dev Mode
+
+If you do not want to generate and configure a public and private key pair, the {extension-name} extension automatically provides an RSA-256 key pair for you in Dev Mode.
```

Comment:
Is this accurate? My understanding is that it also does that when testing?

So maybe the title should be about keys and the text say `in test and dev mode`?

Small nitpick, we usually don't capitalize `dev mode`.